source-oracle/replication.go

```diff
@@ -204,9 +211,10 @@ func (s *replicationStream) addLogFiles(ctx context.Context, startSCN, endSCN in
 			return fmt.Errorf("adding logfile %q (%s, %d, %s, %s) to logminer: %w", f.Name, f.Status, f.FirstChange, f.DictStart, f.DictEnd, err)
 		}
 	}
+
 	s.redoSequence = redoSequence
 
-	return nil
+	return err
```

Comment:
Can `err` ever not be `nil` here? I'm more used to seeing the `return nil` at the end of a function, and returning `err` makes me think there is some way for it to be set to non-nil somewhere in the function, but I'm not seeing anywhere for that to be the case.

Reply:
don't know why I have `return err` here! it is super confusing, changed it to `return nil` 👍🏽